graph-kiota/kiota.md

```diff
@@ -0,0 +1,180 @@
+### What is Kiota? 
+Learn [Kiota](https://github.com/microsoft/kiota#readme)
+
+### APIs that can use Kiota 
+- Any HTTP API with an Open API specification (For generators). Not limited to Microsoft Graph.
+
+### Kiota libraries
+
+Common for all languagues:
+- Kiota generator: Generates the request builders
+The generated code contains the APIClient class and the request builders. 
```

Comment:
I'd rephrase to request builders for fluent API and models. Api Client is just a special request builder.